docs/performance.md

```diff
@@ -0,0 +1,74 @@
+# Measuring performance impact
+
+## Summary
+
+The applications being audited are attached an eBPF program, impact on
```

Comment:
`are attached an eBPF program, which impacts performance.` instead of `are attached an eBPF program, impact on performance cannot be ignored.`